types/enzyme/index.d.ts

```diff
@@ -363,6 +364,8 @@ export interface CommonWrapper<P = {}, S = {}, C = Component<P, S>> {
     length: number;
 }
 
+export type Parameters<T> = T extends (...args: infer A) => any ? A : never;
```

Comment:
We should be able to remove this now that we're on TS version 3.1 👍

Reply:
Does not work :confused: You can see the output in the last Travis build.

Reply:
Oh, interesting. The inlined version doesn't have a function constraint for the type parameter like the built-in version does. This is actually important for our case. Let's add the inlined helper back and we'll be ready to ship. Sorry for the back and forth!

Reply:
No problem ☺️ I pushed a new version with the inline helper back again 😉